cylc/flow/remote.py

```diff
@@ -162,7 +162,7 @@ def get_includes_to_rsync(rsync_includes=None):
 
 
 def construct_rsync_over_ssh_cmd(
-        src_path, dst_path, platform, rsync_includes=None):
+        src_path, dst_path, platform, rsync_includes=None, badhosts=None):
```

Comment:
I can't find any tests for this method. I guess it should have some, but I think that this is out of scope for this ticket.